tracim/tracim/tests/__init__.py

```diff
@@ -344,3 +352,56 @@ def _create_thread_and_test(self,
                                                parent=folder,
                                                owner=user)
         return thread
+
+
+class TestCalendar(TestController):
+    fixtures = [BaseFixture, TestFixture]
+    application_under_test = 'radicale'
+
+    def setUp(self):
+        super().setUp()
+        self._clear_radicale_fs()
+
+    @staticmethod
+    def _clear_radicale_fs():
+        radicale_fs_path = config.radicale.server.filesystem.folder
+        try:
+            files = os.listdir(radicale_fs_path)
+            for file in files:
+                shutil.rmtree('{0}/{1}'.format(radicale_fs_path, file))
+        except FileNotFoundError:
+            pass  # Dir not exists yet, no need to clear it
+
+    def _get_base_url(self):
+        from tracim.config.app_cfg import CFG
+        cfg = CFG.get_instance()
+
+        return CALENDAR_BASE_URL_TEMPLATE.format(
+            proto='https' if cfg.RADICALE_CLIENT_SSL else 'http',
+            domain=cfg.RADICALE_CLIENT_HOST or '127.0.0.1',
+            port=str(cfg.RADICALE_CLIENT_PORT)
+        )
+
+    def _get_user_calendar_url(self, user_id):
```

Comment:
this code should be in radical integration stuff, not in tests (it may be usefull somewere else in the code)